client/components/site-stats-sticky-link/index.jsx

```diff
@@ -36,11 +39,23 @@ var SiteStatsStickyLink = React.createClass( {
 		}
 	},
 
+	preload: function() {
```

Comment:
I'm pondering what it would look, though. Maybe down the road we actually have a `<Link>` component to handle this.